gammapy/scripts/ts_image.py

```diff
@@ -0,0 +1,89 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from ..utils.scripts import get_parser
+import os
+
+__all__ = ['ts_image']
+
+
+def main(args=None):
+    parser = get_parser(ts_image)
+    parser.add_argument('input', type=str,
+                        help='Input data FITS file name')
+    parser.add_argument('--folder', type=str, default='gaussian2d',
+                        help='Output folder name.')
+    parser.add_argument('--psf', type=str, default='psf.json',
+                        help='JSON file containing PSF information. ')
+    parser.add_argument('--morphology', type=str, default='Gaussian2D',
+                        help="Which source morphology to use for TS calculation."
+                        "Either 'Gaussian2D' or 'Shell2D'.")
+    parser.add_argument('--width', type=float, default=None,
+                        help="Width of the shell, measured as fraction of the"
+                        " inner radius.")
+    parser.add_argument('--scales', type=float, default=[0], nargs='+',
+                        help='List of scales to compute TS maps for in deg.')
+    parser.add_argument('--downsample', type=str, default='auto',
+                        help="Downsample factor of the data to obtain optimal"
+                        " performance."
+                        "Must be power of 2. Can be 'auto' to choose the downsample"
+                        "factor automatically depending on the scale.")
+    parser.add_argument('--residual', action='store_true',
+                        help="Whether to compute a residual TS image. If a residual"
+                        "TS image is computed an excess model has to be provided"
+                        "using the '--model' parameter.")
+    parser.add_argument('--model', type=str,
+                        help='Input excess model FITS file name')
+    parser.add_argument('--threshold', type=float, default=None,
+                        help="Minimal required initial (before fitting) TS value,"
+                        " that the fit is done at all.")
+    parser.add_argument('--overwrite', action='store_true',
+                        help='Overwrite output files.')
+    args = parser.parse_args()
+    ts_image(**vars(args))
+
+
+def ts_image(input, psf, model, scales, downsample, residual, morphology,
```

Comment:
`input` is a Python keyword ... you shouldn't use it for variable names.
Maybe use `data` here or `data_filename`?